### PR TITLE
[issue 58] Webhook delivery shows > for quote formatting when description or check-in comment is blank 

### LINKED ISSUE #58: Webhook delivery shows `>` for quote formatting when description or check-in comment is blank
When Bitbucket users leave a comment or description blank and a subscription delivers this the `>` which is the formatting character for quoted text is still shown even when there is no text to show.
image

Steps:
- Setup Bitbucket
- Create a subscription for PRs in a repo
- Go to the repo and create a new PR
- Leave the check-in comment blank

**Observed** There is the `>` character to show quoted text but, no text.

### PR DESCRIPTION
**Pull request by sibasankarnayak**
fixed of post which shows `>` if description is blank in create and update issue

ticket here

Fixes #58

### CODE REVIEW

File: server/templaterenderer/template.go [hunk 1]
```diff
@@ -77,6 +77,9 @@ func (tr *templateRenderer) init() {
 	var funcMap = sprig.TxtFuncMap()
 	// Quote the body
 	funcMap["quote"] = func(body string) string {
+		if body == "" {
+			return body
```
Comment (hanzei) [inline server/templaterenderer/template.go]: It might be easier to read 
```suggestion
			return ""
```

**Review by hanzei — APPROVED** [review]
LGTM :+1:

**Comment by hanzei** [comment]
@DHaussermann Can we test this PR as part of release testing?

**Review by DHaussermann — APPROVED** [review]
This task can be tested post merge as part of release testing.

LGTM!

**Pull request merged**